x-pack/legacy/plugins/canvas/public/components/page_preview/page_controls.js

```diff
@@ -27,15 +31,19 @@ export const PageControls = ({ pageId, onDelete, onDuplicate }) => {
     >
       <EuiFlexItem grow={false}>
         <EuiToolTip content="Clone">
```

Comment:
This `content` prop probably needs to be translated as well while you're here. It's where the content of of tooltip popover thingy comes from 

![image](https://user-images.githubusercontent.com/515272/65903837-20b1e700-e38b-11e9-9787-dfe3b4bf79c7.png)